scenes/game/local_game.gd

```diff
@@ -5870,6 +5907,7 @@ func begin_extra_attack(events, performing_player : Player, card_id : int):
 
 	# Other relevant passives that need to be undone.
 	# If other extra attack characters are added, consider what else might be needed.
+	# For example: If an extra attack character can have positive power bonus boosts, those need to be removed too.
```

Comment:
thankfully I think dan is the only other one in 3-7 and he doesn't seem to care about positive power